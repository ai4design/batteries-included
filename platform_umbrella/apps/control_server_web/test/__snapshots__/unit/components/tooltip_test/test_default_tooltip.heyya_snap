<div id="important">Hoverable</div>
<div class="hidden" id="tooltip-important" phx-hook="Tooltip" data-target="important">
  <div class="min-w-24 bg-white shadow-sm p-2 rounded-md">
    This is the tootip for something important
  </div>
</div>