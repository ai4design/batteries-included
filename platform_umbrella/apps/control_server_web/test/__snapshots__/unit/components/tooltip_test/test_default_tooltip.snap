"<div id=\"important\">Hoverable</div>\n<div class=\"hidden\" id=\"tooltip-important\" phx-hook=\"Tooltip\" data-target=\"important\">\n  <div class=\"min-w-24 bg-white shadow-sm p-2 rounded-md\">\n    This is the tootip for something important\n  </div>\n</div>"